server/@types/notifyClient/index.d.ts

```diff
@@ -0,0 +1,11 @@
+declare module 'notifications-node-client' {
+  export default class NotifyClient {
+    constructor(url: string, clientKey: string): NotifyClient
```

Comment:
this isn't quite right.  I get the error:
```
[Node] TypeError: notifications_node_client_1.default is not a constructor
[Node]     at loginRouter (/Users/peter.phillips/work/check-my-diary/dist/server/routes/login.js:32:26)
```
when the application now runs.  Not sure if I should have declared an interface instead?

Reply:
@petergphillips I've replied, did a PR and pulled the type file someone had done in a PR on the project itself, seems to work 👍

Reply:
this will all get removed soon anyway ?